TI/Contact.html

```diff
@@ -0,0 +1,116 @@
+              <h3 class="text-center mb-4">Contact Us</h3>
+              <form action="" method="post">
+                <div class="form-group">
+                  <label for="name">Name:</label>
+                  <input type="text" class="form-control" id="name" name="name" required>
+                </div>
+                <div class="form-group">
+                  <label for="email">Email:</label>
+                  <input type="email" class="form-control" id="email" name="email" required>
+                </div>
+                <div class="form-group">
+                  <label for="message">Message:</label>
+                  <textarea class="form-control" id="message" name="message" rows="5" required></textarea>
+                </div>
+                <button type="submit" class="btn btn-primary">Send</button>
+              </form>
+            </div>
+          </div>
+        </div>
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+          integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+          crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+          integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNS0tx"
+          crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+          integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+          crossorigin="anonymous"></script>
+          <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-12">
+                    <div class="map-container" style="margin: 0 auto;">
+                        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d5457.404113691217!2d-71.28456667251032!3d46.84955484070294!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4cb8bd6267898679%3A0xf8093f1dd99352b5!2sCondos%20LB9%20Phase%202!5e0!3m2!1sfr!2sca!4v1682538561290!5m2!1sfr!2sca" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+                    </div>
+                </div>
+            </div>
+        </div>
+        
+        <style>
+            .map-container {
+                position: relative;
+                overflow: hidden;
+                padding-top: 56.25%;
+            }
+            .map-container iframe {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
+            }
+        </style>
+</body>
+<footer class="bg-light">
+    <div class="container py-3">
+        <div class="row">
+            <div class="col-md-6">
+                <a class="nav-link" href="https://www.facebook.com/profile.php?id=100016978482689">Facebook</a>

```

Comment:
Usually, external links must use target='_blank'. This way, the external page will be opened in a new tab, not replacing your current page.